pkgs/top-level/all-packages.nix

```
@@ -23985,6 +23985,8 @@ with pkgs;
 
   keepass-otpkeyprov = callPackage ../applications/misc/keepass-plugins/otpkeyprov { };
 
+  kerbrute = python3Packages.callPackage ../tools/security/kerbrute { };
```

it's discouraged to pass the packageset, please use the interpret
```suggestion
  kerbrute = callPackage ../tools/security/kerbrute { };
```